Gems/Atom/RPI/Code/Source/RPI.Public/Pass/Pass.cpp

```diff
@@ -469,95 +479,138 @@ namespace AZ
                 return;
             }
 
+            // Slot type mask used to skip connections at various stages of initialization
             uint32_t bindingMask = (1 << uint32_t(localBinding->m_slotType));
             if (!(bindingMask & slotTypeMask))
             {
                 return;
             }
 
+            // -- Local Variables --
+
             Name connectedPassName = connection.m_attachmentRef.m_pass;
             Name connectedSlotName = connection.m_attachmentRef.m_attachment;
-            Ptr<Pass> connectedPass;
+            Ptr<PassAttachment> attachment = nullptr;
             PassAttachmentBinding* connectedBinding = nullptr;
             bool foundPass = false;
+            bool slotTypeMismatch = false;
+
+            // -- Search This Pass --
 
             if (connectedPassName == PassNameThis)
             {
                 foundPass = true;
-                const Ptr<PassAttachment> attachment = FindOwnedAttachment(connectedSlotName);
+                attachment = FindOwnedAttachment(connectedSlotName);
+
                 AZ_RPI_PASS_ERROR(attachment, "Pass::ProcessConnection - Pass [%s] doesn't own an attachment named [%s].",
                     m_path.GetCStr(),
                     connectedSlotName.GetCStr());
-                localBinding->SetAttachment(attachment);
-                localBinding->m_originalAttachment = attachment;
-                return;
             }
 
-            if (m_parent)
+            // -- Search Pipeline --
+
+            if (connectedPassName == PipelineGlobalKeyword)
             {
-                if (connectedPassName == PassNameParent)
+                AZ_RPI_PASS_ERROR(m_pipeline != nullptr, "Pass::ProcessConnection - Pass [%s] references pipeline attachment [%s] doesn't have a valid pipeline pointer",
+                    m_path.GetCStr(),
+                    connectedSlotName.GetCStr());
+
+                foundPass = true;   // Using the "Pipeline" keyword, no need to continue searching for passes
+
+                if (m_pipeline)
                 {
-                    foundPass = true;
+                    const PipelineGlobalBinding* globalBinding = m_pipeline->GetPipelineGlobalConnection(connectedSlotName);
+                    if (globalBinding)
+                    {
+                        connectedBinding = m_pipeline->GetPipelineGlobalConnection(connectedSlotName)->m_binding;
+                    }
```

Comment:
Do you need to add a slotTypeMismatch check here?

Reply:
Can't tell if the connection is coming from a parent or neighbor, so can't test correctly for this